keyboards/crkbd/keymaps/benrestech/keymap.c

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2019 @foostan
+Copyright 2020 Drashna Jaelre <@drashna>
+Copyright 2022 Ben Allen <@benrestech>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define KC_NP KC_NO // key is not present
+#define KC_NA KC_NO // present but not available for use
+#define KC_NU KC_NO // available but not used
+
+// non-KC_ keycodes
+#define KC_RST QK_BOOT
+#define KC_TOG RGB_TOG
+#define KC_MOD RGB_MOD
+#define KC_HUI RGB_HUI
+#define KC_SAI RGB_SAI
+#define KC_VAI RGB_VAI
```

Comment:
Why not just use these directly in the keymap?

Reply:
My setup is originally based on the excellent Miryoku layout by @manna-harbour, which is where that came from. I then tweaked it to apply the layout changes that I wanted for my workflow.
I'll update it 👍

Reply:
Updated to use native codes rather than re-mapping.